bower.json

```diff
@@ -25,7 +25,8 @@
     "nunjucks": "nunjucks#1.0.5",
     "nwmatcher": "nwmatcher#1.3.4",
     "pikaday": "~1.3.2",
-    "react": "#0.13.0",
+    "react": "#0.13.3",
```

Comment:
Well at first I did it to see if it would fix the problems with that tab library I was using, but now I'm thinking, why not just use the latest version possible? What's the convention on updating deps in SUMO?